README.md

```diff
@@ -1,2 +1,140 @@
 # model-zoo
 Zoo of BIDS Stats Models (extraordinarily experimental)
+
+
+## How to edit datasets (with small files)
+
+### step 1: fork model-zoo to your profile
+go to [model-zoo](https://github.com/bids-standard/model-zoo) and press the fork button on github and select the profile/organization you would like to fork to.
+
+![fork](.media/fork.png)
+
+### step 2: fork the dataset you need to edit
+Find the dataset you need to edit within model-zoo
+
+![submodule](.media/dataset_submodule.png)
+
+Click on the hyperlink to go to the github repo
+
+![dataset](.media/openneuro_dataset.png)
+
+Click on the fork button and select the profile/organization you would like to fork to.
+
+![forked_dataset](.media/forked_openneuro_dataset.png)
+
+### step 3: clone the forked model-zoo to your local machine
+
+#### template:
+```
+git clone https://github.com/<username>/model-zoo.git
+```
+
+#### example:
+```
+git clone https://github.com/jdkent/model-zoo.git
+```
+
+### step 3b: (optional) change branch in model-zoo
+
+#### template:
+`git checkout -b <branch>`
+
+##### example:
+`git checkout -b add_new_model`
+
+
+### step 4: change directories to the dataset you want to edit
+
+#### template:
+`cd model-zoo/ds<num>/ds<num>`
+
+#### example:
+`cd model-zoo/ds005/ds000005`
+
+### step 5: change the remote dataset to your fork of the dataset
+
+#### template:
+```
+git remote rename origin upstream
+git remote add origin https://github.com/<username>/ds000005.git
+```
+
+#### example:
+```
+git remote rename origin upstream
+git remote add origin https://github.com/jdkent/ds000005.git
+```
+
+### step 6: change the .gitmodules file to your fork of the dataset
+
+#### example:
+open the model-zoo/.gitmodules file in your favorite editor (e.g., vim, vscode, nano)
+
+##### before:
+```
+[submodule "ds005/ds000005"]
+	path = ds005/ds000005
+	url = https://github.com/OpenNeuroDatasets/ds000005
+	datalad-id = b2c7f086-8704-11e8-9916-0242ac120015
+	datalad-url = https://github.com/OpenNeuroDatasets/ds000005
+```
+
+##### after:
+```
+[submodule "ds005/ds000005"]
+        path = ds005/ds000005
+        url = https://github.com/jdkent/ds000005
+        datalad-id = b2c7f086-8704-11e8-9916-0242ac120015
+        datalad-url = https://github.com/jdkent/ds000005
```

Comment:
since ATM all of them (besides one fmriprep'ed from @effigies ) point to original OpenNeuroDatasets, why not to create a dedicated organization (e.g. `bids-model-zoo`) add all contributors, and keep those clones all nicely organized there instead of spreading them all across personal forks?